resources/quality/hms434/abs-fr/hms434_0.4_absfr_high.inst.cfg

```diff
@@ -0,0 +1,37 @@
+[general]
+version = 4
+name = High
+definition = hms434
+
+[metadata]
+setting_version = 8
+type = quality
+quality_type = high
+weight = 1
+material = abs-fr_175
+variant = 0.4mm TP extruder
+
+[values]
+
+wall_0_wipe_dist = 0.0
+ironing_enabled = true
+
+infill_overlap_mm = -0.1
+infill_wipe_dist = 0
+
+default_material_print_temperature = 255
+material_standby_temperature = 160
+default_material_bed_temperature = 115
```

Comment:
Seems a bit hot for ABS! Fine with me of course if you think this prints well, but I wanted to point this out in case you made a mistake.

Reply:
This is a special ABS that prints very well at 255